champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/sword/Concussion.java

```diff
@@ -88,6 +90,11 @@ public void onDamage(CustomDamageEvent e) {
             damagee.addPotionEffect(new PotionEffect(PotionEffectType.BLINDNESS, (int) (level * durationPerLevel) * 20, 0));
             UtilMessage.simpleMessage(damager, getName(), "You gave <alt>" + damagee.getName() + "</alt> a concussion.");
             UtilMessage.simpleMessage(damagee, getName(), "<alt>" + damager.getName() + "</alt> gave you a concussion.");
+
+            Location loc = damagee.getLocation();
+            loc.setYaw(loc.getYaw() + 180);
+            damagee.teleport(loc);
```

Comment:
See if you can use [`Entity#setRotation(float, float)`](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/Entity.html#setRotation(float,float)) because teleporting can allow people to phase or glitch through blocks if they're right next to a wall.

Reply:
This function doesn't work for players unfortunately https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/Entity.html#setRotation(float,float)

Reply:
Alright then